README.md

```diff
@@ -59,4 +63,4 @@ To deploy the backend to Heroku:
 local postgres database has data and works when you run the app locally,
 then run `heroku pg:push LOCAL_DB_NAME postgres`
 
-- Visit the [application](http://text-thresher.herokuapp.com/api/) to make sure it worked.
\ No newline at end of file
+- Visit the [application](http://text-thresher.herokuapp.com/api/) to make sure it worked.
```

Comment:
Good question. There is no 0x0a EOL character at the end of the previous file as there should be to terminate the last line, and vim conforms with Unix behavior and adds the EOL character.

`tail -1 README.md | hd` to confirm.

For the fanciest answer, see the third rated answer referencing POSIX in this stackoverflow question:
[Why should text files end with a newline?](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) 

Mac and PC text editors will output files without the final 0x0a, so we will probably be seeing more of these identical looking last line diffs as my editor goes about tacking on that last 0x0a.